HeterogeneousCore/SonicTriton/src/TritonService.cc

```diff
@@ -238,7 +247,67 @@ void TritonService::preBeginJob(edm::PathsAndConsumesOfModulesBase const&, edm::
   if (fallbackOpts_.wait >= 0)
     fallbackOpts_.command += " -w " + std::to_string(fallbackOpts_.wait);
   for (const auto& [modelName, model] : unservedModels_) {
-    fallbackOpts_.command += " -m " + model.path;
+    //We need to edit the triton config files, need to copy from cvmfs
```

Comment:
I would actually suggest to handle the copying and editing of model config files in the `cmsTriton` script (by passing in the `numberOfThreads` value when the script is called). Two reasons:
1. most operations here are shell commands, therefore more naturally suited to be implemented in bash
2. allows this functionality (controlling number of instances per model) to be used outside of CMSSW/`TritonService`

Reply:
Ah yes good suggestion, thank you!  Let me switch that over